metrics/metrics.go

```diff
@@ -366,6 +366,7 @@ func VerifyMetricsMatch(expectedMetrics, actualMetrics string, metricToIgnore st
 }
 
 func (cmm *csiMetricsManager) registerMetrics() {
+	metrics.RegisterProcessStartTime(cmm.registry.Register)
```

Comment:
Can you add a comment explaining why the start time metric is needed?

Reply:
My understanding of https://github.com/kubernetes/component-base/blob/ea4d5eb7ec6256fe0510de29e86ca44f1ac219d2/metrics/processstarttime.go#L37-L48 is that this function should be called once per process.

Calling it here in a low-level utility functions looks wrong to me. It assumes that `registerMetrics()` is called close to the process start. What if the process combines different csiMetricsManager instances under a single endpoint (this is possible, by setting different subsystems)? Won't the two metrics values conflict?

Reply:
A better place may be in https://github.com/kubernetes-csi/csi-lib-utils/blob/fb645ec6d10e04a2669c2b6db8c92dac17930133/metrics/metrics.go#L323-L344